experimental/add_control_region.py

```diff
@@ -77,5 +81,5 @@ def create_new_trees(input_file, suffix = ''):
         shutil.copy(input_folder + fileToUse, fileToUse)
 
         create_new_trees(fileToUse, suffix=suffix)
-    subprocess.call(['hadoop', 'fs', '-rm', output_folder + fileToUse])
-    subprocess.call(['hadoop', 'fs', '-copyFromLocal', fileToUse, output_folder + fileToUse])
+        subprocess.call(['rm', '/hdfs/'+output_folder + fileToUse])
```

Comment:
Please revert it back to hadoop command and add the ```-skipTrash``` option instead:
https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/FileSystemShell.html#rm

BTW: The trash does not work because you do not have a 'home folder' on HDFS.